### DISCUSSION TITLE
Remove HTML built-in dependencies

### DISCUSSION THREAD

**Discussion by tvroylandt** [quarto-dev/quarto-cli#6812] [Q&A] [answered]
### Description

Hi,

Is there a way to remove the JS/CSS dependencies like `clipboard`, `anchor`, `pupper`, etc... ?

I managed to remove all the Bootstrap part with `theme:none` but not this one.

Thanks !

**Comment by cscheid** [comment, marked as answer]
This should be what you want: https://quarto.org/docs/output-formats/html-basics.html#minimal-html

**Comment by tvroylandt** [comment]
Thanks, that's perfect !
